sources/packages/backend/apps/db-migrations/src/sql/StudentAssessments/Add-previous-date-changed-reported-assessment-id-col.sql

```diff
@@ -0,0 +1,6 @@
+ALTER table
+  sims.student_assessments
+ADD
+  COLUMN previous_date_changed_reported_assessment_id INT REFERENCES sims.student_assessments(id);
+
+COMMENT ON COLUMN sims.student_assessments.previous_date_changed_reported_assessment_id IS 'Last reported assessment id corresponding to the offering date change.';
```

Comment:
What about `Assessment that changed the study dates of the application after previously reported assessment.` ?

Reply:
Thanks for updating the description further as my suggested one was not accurate.